config/nginx.sample.conf

```diff
@@ -192,6 +192,11 @@ server {
       break;
     }
 
+    location /letter_avatar_proxy {
+      rewrite /letter_avatar_proxy/(.*)$ /$1 break;
+      proxy_pass https://avatars.discourse.org;
+    }
```

Comment:
By default I expect this to pass on all client sent headers, plus X-Forwarded-For and so on so I think it has negligible effect privacy wise.